eng/test-build-correctness.ps1

```diff
@@ -67,5 +76,11 @@ catch [exception] {
   exit 1
 }
 finally {
+  if ($enableDumps) {
+    $key = "HKLM:\\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps"
+    Remove-ItemProperty -Path $key -Name 'DumpType'
```

Comment:
Should we remember the original values instead?
Also, I'm not sure what level of isolation we get in Azure. Do we risk impacting other builds by changing global/machine settings?